scenario_type: 1
unity_x_coordinate: 1161.9
unity_z_coordinate: 1278.5

local_origin_latitude: 63.4351667
local_origin_longitude: 10.3929167

